app/core/core-design-system/src/main/kotlin/com/hedvig/android/core/designsystem/material3/HedvigMaterial3Theme.kt

```diff
@@ -130,6 +132,8 @@ internal fun lightHedvigColorScheme(hedvigTonalPalette: HedvigTonalPalette) = He
   onContainedButtonContainer = hedvigTonalPalette.greyscale1000,
   secondaryContainedButtonContainer = hedvigTonalPalette.greyscale100,
   onSecondaryContainedButtonContainer = hedvigTonalPalette.greyscale1000,
+  alwaysBlackContainer = hedvigTonalPalette.greyscale900,
```

Comment:
Should be grayscale1000 according to design, for both light and dark

Reply:
Yeap it is 1000 instead of 900, fixing that.
I posted a comment here https://hedviginsurance.slack.com/archives/C03U9C6Q7TP/p1702551037977529 regarding dark mode because I was not so sure that 1000 is at visible with the dark background. And iOS does this differently too